src/librustc_ast/attr/mod.rs

```diff
@@ -21,55 +21,60 @@ use log::debug;
 use std::iter;
 use std::ops::DerefMut;
 
-pub struct Globals {
+// These are global within a compilation session, but not truly global. They are
```

Comment:
```suggestion
// These are global within a compilation session, not to the process. They are
```

what is "truly global" anyway...